pkgs/by-name/st/stl-thumb/package.nix

```
@@ -0,0 +1,62 @@
+{
+  lib,
+  pkgs,
+  stdenv,
+  fetchFromGitHub,
+  libX11,
+  libXcursor,
+  libXrandr,
+  libXi,
+}:
+pkgs.rustPlatform.buildRustPackage rec {
+  pname = "stl-thumb";
+  version = "0.5.0-6bc9057";
+
+  src = fetchFromGitHub {
+    owner = "unlimitedbacon";
+    repo = pname;
```

```suggestion
    repo = "stl-thumb";
```
Avoid interpolating `pname`.